core/tests/test_utils.py

```diff
@@ -1493,26 +1488,20 @@ def swap_with_call_counter(
             raises: Exception|None. The exception raised by the swapped
                 function. If None, then no exception is raised.
             returns: *. The return value of the swapped function.
-            call_through: bool. Whether to call through to the real function,
```

Comment:
I don't mind this getting dropped, but was just curious about why?

Reply:
It wasn't being used, so the coverage checks flagged it as not being covered by the tests. Rather than write tests for it or write tests that use it, I dropped it since it wasn't used